src/main/java/uk/gov/hmcts/reform/prl/controllers/citizen/HearingsNotificationController.java

```diff
@@ -0,0 +1,172 @@
+@Slf4j
+@RestController
+@SecurityRequirement(name = "Bearer Authentication")
+public class HearingsNotificationController {
+
+    @Autowired
+    private CoreCaseDataApi coreCaseDataApi;
+
+    @Autowired
+    private ObjectMapper objectMapper;
+
+    @Autowired
+    private EmailService emailService;
+
+    @Autowired
+    private CaseService caseService;
+
+    @Autowired
+    private IdamClient idamClient;
+
+    @Value("${citizen.url}")
+    private String hearingDetailsUrl;
+
+    @PostMapping(path = "/hearing-details-notification", produces = APPLICATION_JSON_VALUE, consumes = MULTIPART_FORM_DATA_VALUE)
+    @Operation(description = "Send a notification to the parties about the hearing date.")
+    @ApiResponses(value = {
+        @ApiResponse(responseCode = "200", description = "Notification Sent"),
+        @ApiResponse(responseCode = "400", description = "Bad Request"),
+        @ApiResponse(responseCode = "500", description = "Internal server error")})
+    public String sendHearingNotification(@RequestHeader(HttpHeaders.AUTHORIZATION) String authorisation,
+                                          @RequestHeader("serviceAuthorization") String s2sToken,
+                                          @ModelAttribute HearingDetailsRequest hearingDetailsRequest) throws Exception {
+
+        String caseId = hearingDetailsRequest.getCaseId();
+        CaseDetails caseDetails = coreCaseDataApi.getCase(authorisation, s2sToken, caseId);
+        CaseData caseData = CaseUtils.getCaseData(caseDetails, objectMapper);
+        String eventId = hearingDetailsRequest.getEventId();
+        String newState;
+        caseData = caseData.toBuilder()
+            .state(State.CASE_HEARINGS)
+            .build();
+
+        StartEventResponse startEventResponse =
+            coreCaseDataApi.startEventForCitizen(
+                authorisation,
+                s2sToken,
+                idamClient.getUserInfo(authorisation).getUid(),
+                JURISDICTION,
+                CASE_TYPE,
+                caseId,
+                eventId
+            );
+
+        CaseDataContent caseDataContent = CaseDataContent.builder()
+            .eventToken(startEventResponse.getToken())
+            .event(Event.builder()
+                       .id(startEventResponse.getEventId())
+                       .build())
+            .data(caseData).build();
+
+        coreCaseDataApi.submitEventForCitizen(
+            authorisation,
+            s2sToken,
+            idamClient.getUserInfo(authorisation).getUid(),
+            JURISDICTION,
+            CASE_TYPE,
+            caseId,
+            true,
+            caseDataContent
+        );
+        sendHearingDetailsEmailToCitizen(hearingDetailsRequest, caseData);
+
+        return "SUCCESS";
+    }
+
+    private void sendHearingDetailsEmailToCitizen(HearingDetailsRequest hearingDetailsRequest, CaseData caseData) {
+
+        String email = "";
+        String  partyId = hearingDetailsRequest.getPartyId();
```

Comment:
this email is supposed to be for all the parties, not anyone specific. You wont get the partyId in request